Wsl.psd1

```diff
@@ -9,7 +9,7 @@
 RootModule = 'Wsl.psm1'
 
 # Version number of this module.
-ModuleVersion = '2.0.0'
+ModuleVersion = '2.0.1'

```

Comment:
This module uses semver, so this should be 2.1.0 since it adds new functionality.
```suggestion
ModuleVersion = '2.1.0'
```

Reply:
Updated.